It would seem our documentation is inaccurate BA-5828 (#5074)
broadinstitute/cromwell: docs/api/RESTAPI.md

|Type|Name|Description|Schema|Default|
|---|---|---|---|---|
|**Path**|**version**  <br>*required*|Cromwell API Version|string|`"v1"`|

> |**Query**|**additionalQueryResultFields**  <br>*optional*|Currently only 'labels' and `parentworkflowId` are valid values here. Use it to include a list of labels and/or parent workflow IDs with each result.|< string > array(multi)||

Reviewer: Is that capitalization really correct for `parentworkflowId `...? 😨

Reviewer: In the other file this is written as `parentWorkflowId `

Author: @cjllanwarne Yeah duh. I just did a global search for text files which had this, forgot about the autogeneration

> |**Query**|**additionalQueryResultFields**  <br>*optional*|Currently only 'labels' and `parentworkflowId` are valid values here. Use it to include a list of labels and/or parent workflow IDs with each result.|< string > array(multi)||

Reviewer 2: ```suggestion
|**Query**|**additionalQueryResultFields**  <br>*optional*|Currently only 'labels' and `parentWorkflowId` are valid values here. Use these to include a list of labels and/or parent workflow ID with each result.|< string > array(multi)||
```
